smartmin/models.py

```diff
@@ -222,7 +222,7 @@ def import_xls(cls, filename, user, import_params, log=None, import_results=None
         naive_timezone = (
             zoneinfo.ZoneInfo(import_params["timezone"])
             if import_params and "timezone" in import_params
-            else timezone.utc
```

Comment:
That was an alias to datetime.timezone.utc and that is removed in django 5.0

Reply:
`USE_DEPRECATED_PYTZ` is also removed as well

Reply:
I had tried that in https://github.com/nyaruka/smartmin/pull/179/commits/a54199de6164529c9665d0a0436439bc161ccb75 but I did not like that since I had to alias `datetime.timezone` to not collide with   `django.utils.timezone` still used for things like `timezone.now()`

Reply:
And zoneinfo is the recommended way in the release notes
https://docs.djangoproject.com/en/5.0/releases/4.0/#zoneinfo-default-timezone-implementation

Reply:
Yeah that is annoying... I don't see anywhere tho where it says `zoneinfo.ZoneInfo("UTC")` is recommended

What if we imported `now` like...

```
from django.utils.timezone import now
```

Reply:
Okay

Reply:
Or 

```
from django.utils import timezone as utils_timezone
```?

Reply:
@rowanseymour I found that we can use 
```
from django.utils import timezone
```

and then use the variable `timezone.timezone.utc`

since utils timezone import datetime timezone

https://github.com/django/django/blob/617bcf611f3daa796e4054ba041089ece30a32fc/django/utils/timezone.py#L8

Reply:
Doesn't feel ideal to rely on indirect imports...

Reply:
I just tried `from django.utils.timezone import now` but could also cause problem in 
https://github.com/nyaruka/smartmin/blob/115f821f4ee76623de661c632237e1d1a54f4fda/smartmin/templatetags/smartmin.py#L165

Reply:
Frustratingly there doesn't seem to be much consistency internally in Django as you can see in this diff https://github.com/django/django/commit/bb61f0186d5c490caa44f3e3672d81e14414d33c

Sometimes it's accessed as `datetime.timezone.utc`, sometimes `now` is imported on its own, sometimes its aliased...

Reply:
I think I'm leaning toward always accessing it like `datetime.timezone.utc` and then continuing to use `timezone.now()` like that

Reply:
Well I guess on our side we can just decide to consistently use the alias approach for now 
`from datetime import timezone as datetime_timezone`

Reply:
Yes, I agree we need to keep `timezone.now()` but `datetime.timezone.utc` has issues since we mostly import 
`from datetime import datetime, timedelta`

I think we can use the alias for timezone 
`from datetime import timezone as datetime_timezone`

That will easily for for the existing imports

Reply:
Can we pick something shorter than `datetime_timezone`? `tzone`?

Reply:
Okay, I'll use `tzone`